middleware/realip.go

```diff
@@ -22,7 +22,7 @@ var xRealIP = http.CanonicalHeaderKey("X-Real-IP")
 // RemoteAddr will see the intended value.
 //
 // You should only use this middleware if you can trust the headers passed to
-// you (in particular, the two headers this middleware uses), for example
+// you (in particular, the three headers this middleware uses), for example
```

Comment:
LGTM.

Can we list the header names in this comment explicitly?

Reply:
They are listed on line 17, so there is no need to duplicate here.